### PR TITLE
Updates for learn center

### PR DESCRIPTION
**Pull request by drewdrewthis**

### CODE REVIEW

File: src/atoms/Button/buttons.module.scss [hunk 1]
```diff
@@ -57,191 +11,61 @@ $brand-white: color('neutral-8');
...
   box-shadow 150ms ease-out;
   width: 100%;
   font-size: ru(.5);
+  text-transform: uppercase;
 
   .icon {
     path {
       transition: fill 300ms ease-out;
-      fill: $button-color;
+      fill: $action-color;
```
Comment (jmcolella) [inline src/atoms/Button/buttons.module.scss]: is the `Icon` only supposed to be used on `outline` Buttons? The first example is using the `Icon` on the solid orange button and you can't see it. Can you provide some clarity here/update the Readme?

File: src/atoms/Button/index.js [hunk 2]
```diff
@@ -49,13 +47,18 @@ Button.propTypes = {
   type: PropTypes.string,
 
   /**
-   * Possible button variants are: `info`, `disabled`, `toggle`, `action`, `actionDisabled`, `lowlight`, `outline`, or `button` (default)
+   * Possible button variants are: `info`, `disabled`, `outline` (deprecated)
```
Comment (jmcolella) [inline src/atoms/Button/index.js]: what exactly is deprecated here? You can add a deprecation warning with a PropTypes function for specific keywords if necessary

File: src/molecules/FeatureSquare/index.js [hunk 3]
```diff
@@ -28,10 +29,12 @@ function FeatureSquare(props) {
 
   return (
     <div className={classnames(...classes)}>
-      <Icon icon={icon} className={styles['icon']} />
-      <Text type={4} semibold>{ header }</Text>
-      <Text type={6} className={styles['subheader']}>{ subheader }</Text>
-      <Button onClick={onClick}>{ buttonText }</Button>
+      <Icon icon={icon} className={styles['icon']} width='100' />
```
Comment (jmcolella) [inline src/molecules/FeatureSquare/index.js]: No example for this component?

File: src/organisms/cards/PolicyCard/PolicyActions.js [hunk 4]
```diff
@@ -48,8 +48,6 @@ export const PolicyActions = (props) => {
       <Spacer spacer={3} />
       <Layout largeCols={[ 6 ]} mediumCols={[ 12 ]} smallCols={[ 12, 6, 6 ]} style={{ width: '100%' }}>
         <Button
-          variant='action'
```
[the remark this answers is not in the document]
Comment (drewdrewthis) [reply src/organisms/cards/PolicyCard/PolicyActions.js]: These were removed from the Button component and was causing warnings.

File: src/molecules/FeatureSquare/index.js (hunk 3 above)
Comment (drewdrewthis) [reply src/molecules/FeatureSquare/index.js]: Nahh, moving fast -- can go back and add it.

File: src/atoms/Text/index.js [hunk 5]
```diff
@@ -59,17 +60,17 @@ function Text(props) {
...
     styles[tag],
     styles[align],
     spaced && styles['spaced'],
     italic && styles['italic'],
-    styles[`type-${font}-${type}-bold`],
-    styles[`type-${font}-${type}-medium`],
-    styles[`type-${font}-${type}-regular`],
+    styles[`type-${font}-${size || type}-bold`],
+    styles[`type-${font}-${size || type}-medium`],
+    styles[`type-${font}-${size || type}-regular`],
```
[the remark this answers is not in the document]
Comment (drewdrewthis) [reply src/atoms/Text/index.js]: I'm only adding the size option --  the other stuff was from a previous commit.

File: src/atoms/Text/index.js [hunk 6]
```diff
@@ -59,17 +60,17 @@ function Text(props) {
   const Element = `${tag}`;
   const classes = [
     styles['text'],
-    styles[`typography-${type}`],
+    styles[`typography-${size || type}`],
```
[the remark this answers is not in the document]
Comment (drewdrewthis) [reply src/atoms/Text/index.js]: Makes sense.

File: src/atoms/Button/index.js (hunk 2 above)
Comment (drewdrewthis) [reply src/atoms/Button/index.js]: The "outline" is deprecated.

File: src/atoms/Button/buttons.module.scss (hunk 1 above)
Comment (drewdrewthis) [reply src/atoms/Button/buttons.module.scss]: Fair... I moved through this quickly. We no longer have buttons with Icons, so I just left it.

Comment (drewdrewthis) [reply src/atoms/Button/buttons.module.scss]: Updating to make the icon white.

**Pull request merged**